_playwright-tests/UI/SnapshotRepo.spec.ts

```diff
@@ -91,15 +91,7 @@ test.describe('Snapshot Repositories', () => {
       await edited_row.getByLabel('Kebab toggle').click();
       await page.getByRole('menuitem', { name: 'Delete' }).click();
       await expect(page.getByText('Remove repositories?')).toBeVisible();
-
-      await Promise.all([
-        page.waitForResponse(
-          (resp) =>
-            resp.url().includes('bulk_delete') && resp.status() >= 200 && resp.status() < 300,
-        ),
-        page.getByRole('button', { name: 'Remove' }).click(),
-      ]);
-
```

Comment:
I think this would still be nice to have as without this, it won't assert/verify the response at all 🤔 

> This step is deleting a single repo not doing a bulk delete.
⬆️ since this is the case for this step, why not just check for a `delete` instead of a `bulk_delete` in the URL?

Reply:
I thought Playwright takes care of waiting for a button to be enabled?